src/__tests__/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].test.tsx

```diff
@@ -305,12 +305,12 @@ describe("pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[le
       expect(seo).toEqual(
         expect.objectContaining({
           title:
-            "Lesson: Adverbial complex sentences | AQA | KS2 English | NEXT_PUBLIC_SEO_APP_NAME",
+            "Adverbial complex sentences | AQA KS2 | Y3 English Lesson Resources | NEXT_PUBLIC_SEO_APP_NAME",
```

Comment:
As above, I think we can omit the pipe between the lesson title and the exam board so it would be: "Adverbial complex sentences AQA KS2 | Y3 ... "